### PR TITLE
Do not set SRC_PATH in plan.

### PR DESCRIPTION
**Pull request by chef-delivery**
This Pull Request was opened by Chef Automate user Ryan Hass

This is an attempt to fix the deploy phase in our AURD.

----
Ready to merge? [View this change](https://automate.chef.co/e/chef/#/organizations/CIA/projects/omnitruck/changes/f6534802-c2cb-4928-96bf-a906b9261e8b) in Chef Automate and click the Approve button.

### CODE REVIEW

File: habitat/plan.sh [hunk 1]
```diff
@@ -9,10 +9,6 @@ pkg_license=('Apache-2.0')
 pkg_deps=(
   core/coreutils
 )
-# This is set to force the source path to the root level during automate builds
-# You must build plan from the root dir with command `build habitat`
-# Recommended to execute `scripts/hab-it`
-SRC_PATH="../"
```
Comment (wrightp) [inline habitat/plan.sh]: Are we ignoring our own comments now? :)

Does habitat now continue to search up the dir tree to find the Gemfile?

Comment (rhass) [reply habitat/plan.sh]: I am not sure, but I wanted to roll the dice to see if the changes to the Ruby scaffolding fixed this issue.

Comment (wrightp) [reply habitat/plan.sh]: I think this change is a gamble when we can just look at the hab project. Also, this is testable locally. Is there a reason you are submitting a change through automate for this?

Comment (rhass) [reply habitat/plan.sh]: Just fixed one thing, but works locally (now). From the comment I assumed it was only an issue when running this via automate/workflow.

Comment (wrightp) [reply habitat/plan.sh]: Let's pair on this and maybe it will jug our memory. That work was all done in haste.

Comment (tduffield) [reply habitat/plan.sh]: Just as a note, I'm 99% sure you still need this to make it work with habitat_build

**Pull request merged**